csharp/src/Ice/OutgoingRequestFrame.cs

```diff
@@ -71,6 +71,9 @@ public IList<ArraySegment<byte>> Payload
         /// <summary>The frame byte count.</summary>
         public int Size { get; private set; }
 
+        // Whether or not to Compress the content of the frame with Ice1
+        internal bool Compress;
```

Comment:
For non-private data members, we generally prefer properties over fields.

It's likely this property should be get-only (set during construction). 

And should it be public?

Reply:
Yes, it should be get only, I changed it to a property.

I didn't make it public because it's a Ice1 property only for now and an implementation detail... it won't be useful for ice2 since the compression is handled by the encapsulation. I guess we could provide it publicly "for info" but I doubt it will be very useful.